test/dlc_tests/ec2/pytorch/training/test_pytorch_training_2_3.py

```diff
@@ -0,0 +1,103 @@
+import pytest
```

Comment:
are gdrcopy/transformerengine running on this version? if so, let's see if we can bring them into one of the serialized test buckets. Can leave EFA alone for now

Reply:
note - this is more of a nit ^^^, it's fine to leave these as is for now if your changes are bringing in testing requirements for PT 2.1 and 2.2 - these types of things should be decoupled